Makefile

```diff
@@ -96,13 +96,14 @@ TARGET := sm64
 
 
 # GRUCODE - selects which RSP microcode to use.
-#   f3dex   -
-#   f3dex2  -
-#   l3dex2  - F3DEX2 version that only renders in wireframe
-#   f3dzex  - newer, experimental microcode used in Animal Crossing
-#   super3d - extremely experimental version of Fast3D lacking many features for speed
-GRUCODE ?= f3dzex
-$(eval $(call validate-option,GRUCODE,f3dex f3dex2 f3dex2pl f3dzex super3d l3dex2))
+#   f3dex   - Upgraded Fast3D. Offers worse performance than Fast3DEX2, but has a more precise z-buffer.
+#   f3dex2  - Upgraded Fast3DEX. Good performance and widely supported across almost all emulators.
+#   l3dex2  - Fast3DEX2 version that only renders in wireframe.
+#   f3dzex  - Newer, experimental microcode based on Fast3DEX2 used in Animal Crossing and Zelda 64.
+#   f3dex3  - Upgraded Fast3DEX2. Great performance, but as of February 4th, 2024, it is only supported across LLE emulators and real hardware.
+#   super3d - Extremely experimental version of Fast3D lacking many features and simplified routines for speed.
+GRUCODE ?= f3dex3
```

Comment:
I'd personally leave this to use f3dzex by default for the time being, until HLE plugins are developed. Using f3dex3 should probably be a conscious choice as to not risk people that don't know what they're doing breaking emulators without a clue in the world of what's going on (discussion welcome idk).

Reply:
This is an accidental change. I will change it to F3DZEX.

Reply:
Corrected.

Reply:
Uncorrected

Reply:
Corrected.